sagemaker-pipelines/nlp/amazon_comprehend_sagemaker_pipeline/prepare_data.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+import os
+import pandas as pd
+from glob import glob
+from tqdm import tqdm
+
+os.system("du -a /opt/ml")
+
+SRCTRAINFILE = glob("/opt/ml/processing/input_train/*.csv")[0]
+print(SRCTRAINFILE)
+SRCTESTFILE = glob("/opt/ml/processing/input_test/*.csv")[0]
+print(SRCTESTFILE)
+
+DSTTRAINFILE = "/opt/ml/processing/train/train.csv"
+DSTTESTFILE = "/opt/ml/processing/test/test.csv"
+
+# Preparation of the train set
+trainFrame = pd.read_csv(SRCTRAINFILE, header=None)
+testFrame = pd.read_csv(SRCTESTFILE, header=None)
+
+# Amazon Comprehend recommend[s] that you train the model with at least 50 training documents for
```

Comment:
I think `Amazon Comprehend recommends...` is fine. Alternatively, `AWS recommends that you train an Amazon Comprehend model...` Or even `We recommend` .